ui/js/dfv/src/pods-dfv.js

```diff
@@ -599,6 +599,52 @@ window.PodsDFV = {
 		return true;
 	},
 
+	/**
+	 * Check if Pod being edited is valid.
+	 *
+	 * Returns array of validation Messages
+	 *
+	 * @param {string} pod         Pod slug/name.
+	 * @param {int}    itemId      Object ID.
+	 * @param {int}    formCounter Form index.
+	 *
+	 * @return {string[]} Array of validation messages
+	 */
+	checkValidation( pod, itemId, formCounter ) {
+		const storeKey = createStoreKey(
+			pod,
+			itemId,
+			formCounter,
+			STORE_KEY_DFV
+		);
+
+		const stored = select( storeKey );
+
+		// Store not found.
+		if ( ! stored ) {
+			return undefined;
+		}
+		//dispatch toggleNeedsValidating
+		dispatch( storeKey ).toggleNeedsValidating();
+		//select getValidationMessages
+		const validationMessages = select( storeKey ).getValidationMessages();
+		console.log( { validationMessages } );
+		return validationMessages;
```

Comment:
We should normalize this to using `const form = this.detectForm( pod, itemId, formCounter )` so it can properly detect in a few different scenarios. Then it can use the returned `form.stored`.

```suggestion
		const form = this.detectForm(
			pod,
			itemId,
			formCounter
		);

		if ( 'undefined' === typeof form ) {
			return undefined;
		}

		// Check validations.
		dispatch( form.storeKey ).toggleNeedsValidating();

		// Get validation messages.
		const validationMessages = form.stored.getValidationMessages();
		
		// Debug output for validation messages for now. @todo Remove this.
		console.log( { validationMessages } );

		return validationMessages;
```